README.md

```diff
@@ -733,126 +771,130 @@ analyzer = DummyAnalyzer()
 <table ><tbody ><tr></tr><tr>
 <td><details ><summary><img style="vertical-align:middle;margin:2px 10px" src="https://raw.githubusercontent.com/obsei/obsei-resources/master/logos/slack.svg" width="25" height="25"><b>Slack</b></summary><hr>
 
- ```python
+```python
 from obsei.sink.slack_sink import SlackSink, SlackSinkConfig
 
 # initialize slack sink config
 sink_config = SlackSinkConfig(
-    # Provide slack bot/app token
-    # For more detail refer https://slack.com/intl/en-de/help/articles/215770388-Create-and-regenerate-API-tokens
-    slack_token="<Slack_app_token>",
-    # To get channel id refer https://stackoverflow.com/questions/40940327/what-is-the-simplest-way-to-find-a-slack-team-id-and-a-channel-id
-    channel_id="C01LRS6CT9Q"
+   # Provide slack bot/app token
+   # For more detail refer https://slack.com/intl/en-de/help/articles/215770388-Create-and-regenerate-API-tokens
+   slack_token="<Slack_app_token>",
+   # To get channel id refer https://stackoverflow.com/questions/40940327/what-is-the-simplest-way-to-find-a-slack-team-id-and-a-channel-id
+   channel_id="C01LRS6CT9Q"
 )
 
 # initialize slack sink
 sink = SlackSink()
 ```
+
 </details>
 </td>
 </tr>
 <tr>
 <td><details ><summary><img style="vertical-align:middle;margin:2px 10px" src="https://raw.githubusercontent.com/obsei/obsei-resources/master/logos/zendesk.png" width="20" height="20"><b>Zendesk</b></summary><hr>
 
- ```python
+```python
 from obsei.sink.zendesk_sink import ZendeskSink, ZendeskSinkConfig, ZendeskCredInfo
 
 # initialize zendesk sink config
 sink_config = ZendeskSinkConfig(
-    # For custom domain refer http://docs.facetoe.com.au/zenpy.html#custom-domains
-    # Mainly you can do this by setting the environment variables:
-    # ZENPY_FORCE_NETLOC
-    # ZENPY_FORCE_SCHEME (default to https)
-    # when set it will force request on:
-    # {scheme}://{netloc}/endpoint
-    # provide zendesk domain
-    domain="zendesk.com",
-    # provide subdomain if you have one
-    subdomain=None,
-    # Enter zendesk user details
-    cred_info=ZendeskCredInfo(
-        email="<zendesk_user_email>",
-        password="<zendesk_password>"
-    )
+   # For custom domain refer http://docs.facetoe.com.au/zenpy.html#custom-domains
+   # Mainly you can do this by setting the environment variables:
+   # ZENPY_FORCE_NETLOC
+   # ZENPY_FORCE_SCHEME (default to https)
+   # when set it will force request on:
+   # {scheme}://{netloc}/endpoint
```

Comment:
These lines are stale now, I forgot to remove it in https://github.com/obsei/obsei/pull/221